cmake/Modules/ObsHelpers_macOS.cmake

```diff
@@ -0,0 +1,305 @@
+				COMMENT "Codesigning Browser Helper ${_HELPER_OUTPUT_NAME}"
+				VERBATIM)
+		endforeach()
+	else()
+		if(TARGET obs-browser-helper)
+			add_custom_command(TARGET ${target} POST_BUILD
+				COMMAND "${CMAKE_COMMAND}" -E copy
+					"$<TARGET_FILE:obs-browser-helper>"
+					"$<TARGET_BUNDLE_CONTENT_DIR:obs>/PlugIns/$<TARGET_FILE_BASE_NAME:obs-browser>/Contents/MacOS/$<TARGET_FILE_NAME:obs-browser-helper>"
+				VERBATIM)
+
+			add_custom_command(TARGET ${target} POST_BUILD
+				COMMAND /bin/sh -c "codesign --force --sign \"${OBS_BUNDLE_CODESIGN_IDENTITY}\" $<$<BOOL:${OBS_CODESIGN_LINKER}>:--options linker-signed >\"$<TARGET_BUNDLE_CONTENT_DIR:obs>/PlugIns/$<TARGET_FILE_BASE_NAME:obs-browser>/Contents/MacOS/$<TARGET_FILE_NAME:obs-browser-helper>\""
+				COMMENT "Codesigning obs-browser-helper"
+				VERBATIM)
+		endif()
+	endif()
+endfunction()
+
+# Helper function to set-up OBS plugins and helper binaries for macOS bundling
+function(setup_obs_modules target)
+	get_property(OBS_MODULE_LIST GLOBAL PROPERTY OBS_MODULE_LIST)
+	foreach(_MODULE IN LISTS OBS_MODULE_LIST)
+		add_custom_command(TARGET ${target} POST_BUILD
+			COMMAND "${CMAKE_COMMAND}" -E copy_directory
+			  "$<TARGET_BUNDLE_DIR:${_MODULE}>"
+			  "$<TARGET_BUNDLE_CONTENT_DIR:${target}>/PlugIns/$<TARGET_FILE_BASE_NAME:${_MODULE}>.plugin"
+			VERBATIM)
+	endforeach()
+
+	install(TARGETS ${OBS_MODULE_LIST}
+		LIBRARY DESTINATION "$<TARGET_FILE_BASE_NAME:${target}>.app/Contents/PlugIns"
+			COMPONENT obs_plugins
+			NAMELINK_COMPONENT ${target}_Development)
+
+	get_property(OBS_MODULE_LIST GLOBAL PROPERTY OBS_SCRIPTING_MODULE_LIST)
+	foreach(_MODULE IN LISTS OBS_MODULE_LIST)
+		add_custom_command(TARGET ${target} POST_BUILD
+			COMMAND "${CMAKE_COMMAND}" -E copy
+			  "$<TARGET_FILE:${_MODULE}>"
+			  "$<TARGET_BUNDLE_CONTENT_DIR:${target}>/PlugIns/$<TARGET_FILE_NAME:${_MODULE}>"
+			VERBATIM)
+
+		if(${_MODULE} STREQUAL "obspython")
+			install(FILES "$<TARGET_FILE_DIR:${_MODULE}>/${_MODULE}.py"
+				DESTINATION "$<TARGET_FILE_BASE_NAME:${target}>.app/Contents/PlugIns"
+				COMPONENT obs_scripting_plugins)
+
+			add_custom_command(TARGET ${target} POST_BUILD
+				COMMAND "${CMAKE_COMMAND}" -E copy
+					"$<TARGET_FILE_DIR:${_MODULE}>/${_MODULE}.py"
+					"$<TARGET_BUNDLE_CONTENT_DIR:${target}>/PlugIns/${_MODULE}.py"
+				COMMAND /bin/sh -c "codesign --force --sign \"${OBS_BUNDLE_CODESIGN_IDENTITY}\" $<$<BOOL:${OBS_CODESIGN_LINKER}>:--options linker-signed >\"$<TARGET_BUNDLE_CONTENT_DIR:${target}>/PlugIns/${_MODULE}.py\""
+				VERBATIM)
+		endif()
+	endforeach()
+
+	install(TARGETS ${OBS_SCRIPTING_MODULE_LIST}
+		RUNTIME DESTINATION "$<TARGET_FILE_BASE_NAME:obs>.app/Contents/PlugIns"
+			COMPONENT obs_scripting_plugins)
+
+	if(TARGET obs-ffmpeg-mux)
+		install(TARGETS obs-ffmpeg-mux
+			RUNTIME DESTINATION "$<TARGET_FILE_BASE_NAME:obs>.app/Contents/MacOS"
+				COMPONENT obs_plugins)
+	endif()
+endfunction()
+
+# Helper function to finalise macOS app bundles
```

Comment:
Nitpick:  finalize

Reply:
🇬🇧🇬🇧🇬🇧🇬🇧


Yeah, fine will change it 🥲